src/sparseml/core/logger/utils/frequency_manager.py

```diff
@@ -16,7 +16,12 @@
 from typing import Literal, Union
 
 
-__all__ = ["FrequencyManager"]
+__all__ = [
+    "FrequencyManager",
+    "PossibleLoggingMode",
```

Comment:
why not just `LoggingMode`?

Reply:
Updated to `LoggingModeType` to avoid confusion